api/repositories/pod_repository.go

```diff
@@ -405,6 +399,21 @@ func (r *PodRepo) GetRuntimeLogsForApp(ctx context.Context, logger logr.Logger,
 	return appLogs, nil
 }
 
+func lineToAppLogRecord(line []byte) LogRecord {
```

Comment:
Extracted this to appease CodeClimate 😅 

To be honest, it was probably right about this function being too long.